clang/test/CodeGen/cheri/riscv/non-cheri-march.c

```diff
@@ -0,0 +1,11 @@
+// NOTE: Assertions have been autogenerated by utils/update_cc_test_checks.py
+/// Using __capability without the appropriate -march string previously
+/// crashed clang during codegen instead of emitting an error
+// RUN: %clang_cc1 -triple riscv64 -o - -emit-llvm %s -verify=no-cheri
```

Comment:
This probably belongs in clang/test/Parser?

Reply:
True, I added it here since it only crashes if you try to emit code. Will move to parser.